<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BShow1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QAddnotes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Note</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Qquit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
